### PR TITLE
Disabling alignment helps minimizing git diffs

### PR DESCRIPTION
**Pull request by nightscape**

### CODE REVIEW

File: readme/Faq.scalatex [hunk 1]
```diff
@@ -64,6 +64,17 @@
        @li
          code is read waaay more often outside of code reviews, for example
          when you are actually coding.
+       @li
+         it helps to disable most alignment options
+         (so that e.g. when you rename the variable with the longest name to become longer,
+         other lines are not impacted):
+         
+         @cliFlags
+            align.tokens = []
```
Comment (olafurpg) [inline readme/Faq.scalatex]: align = none?

Comment (nightscape) [reply readme/Faq.scalatex]: I didn't know that one :wink:

**Comment by nightscape** [comment]
I'm open to where to put this. I thought here is a good place because that section already talks about git diffs.
We could put it under Configuration and then link there from the FAQ?

**Comment by olafurpg** [comment]
Under the `align=none` section we can add a "Pro tip. Enable this setting to minimize git diffs/conflicts from renamings and other refactorings." We can also add a new section/question under FAQ "Which configuration options minimize diffs/conflicts?"

**Pull request merged**